mypy/visitor.py

```diff
@@ -22,220 +22,220 @@ class NodeVisitor(Generic[T]):
 
     # Module structure
 
```

Comment:
These are all  overloaded later with different arg names.  See also needing the positional-only syntax.